src/consumption.cpp

```diff
@@ -162,6 +248,15 @@ edible_rating player::can_eat( const item &food, bool interactive, bool force )
         return INEDIBLE;
     }
 
+    if( comest->comesttype == "FOOD" || comest->comesttype == "DRINK" ) {
```

Comment:
Would be better if you integrated it with the `chew` and `drinkable` variables below. Bring them up here and use them in the `if`.